web/apps/serve.go

```diff
@@ -321,7 +321,13 @@ func (s serveParams) CozyData() ([]byte, error) {
 		},
 		"flags": flags,
 	}
-	return json.Marshal(data)
+	bytes, err := json.Marshal(data)
+
+	if err != nil {
+		panic(err)
```

Comment:
`panic` must only be used as a least resort. Here, we can return the error: `return "", err`

Reply:
I used panic as I saw it in the cozyclientjs function in the same file, should it be changed there also ?

Reply:
Yes, it should be changed: https://github.com/cozy/cozy-stack/pull/2397